livekit-plugins/livekit-plugins-anthropic/livekit/plugins/anthropic/llm.py

```diff
@@ -40,11 +50,18 @@
 )
 
 
+class ToolChoice(TypedDict, total=False):
+    type: Literal["auto", "any", "tool", "none", "required"]
```

Comment:
what is the difference between `tool` and `required`?

What is `any` used for?

Reply:
`required` (OpenAI specific param) - the model must call one or more tools. 
`any` (Anthropic specific param) - the model must call one or more tools. 
`tool` (Anthropic specific param) - the model must call specific tool

Reply:
can we merge `any` and `required`?

Reply:
Yes, we can, but I think we may need to document this otherwise, it might confuse users.